src/internal/js/mod.rs

```diff
@@ -307,10 +313,26 @@ impl JsString {
         }
     }
 
+    pub fn ucs2_value(self) -> Vec<u16> {
```

Comment:
There is no (stable) pointer though, V8 only lets you copy out the character data.